.lsp/config.edn

```diff
@@ -0,0 +1 @@
+{:clean {:ns-inner-blocks-indentation :keep}}
```

Comment:
find this in one clojure-lsp issue https://github.com/clojure-lsp/clojure-lsp/issues/1141
yet it's not in clojure-lsp's [all-available-settings.edn](https://github.com/clojure-lsp/clojure-lsp/blob/master/docs/all-available-settings.edn)

should create an issue for clojure-lsp